app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,52 @@
+class QuestionsController < ApplicationController
+  before_action :find_question, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+    @answer = Answer.new
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+  end
+
+  def create
+    @question = Question.new(question_params)
+
+    if @question.save
+      redirect_to @question, notice: 'Question was successfully created'
+    else
+      render :new
+    end
+  end
+
+  def update
+
+    if @question.update(question_params)
+      redirect_to @question, notice: 'Question was successfully updated'
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @question.destroy
+    redirect_to questions_path, notice: 'Question was deleted'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.